### PR TITLE
Added "leave server" functionality

### PR DESCRIPTION
**Pull request by brandongallagher1999**
Each server card now has the ability to leave the server if they want.

### CODE REVIEW

File: src/components/card.js [hunk 1]
```diff
@@ -25,13 +26,25 @@ const Card = ({ id, name, permissions, icon }) => {
...
+  }
+
   return (
     <CardWrapper>
       <CardImg src={iconBaseURL} alt={name} />
       <TextWrapper>
         <h3>{name}</h3>
         <p>ID: {id}</p>
         <p>Permissions: {permissions}</p>
+        <button onClick = {leaveServer}> Leave Server </button>
```
Comment (chakrakan) [inline src/components/card.js:47]: can you create a styled component and create a simple button style that fits in with the rest? The default one sticks out 😅

**Review by chakrakan — COMMENTED** [review]
LGTM and thanks for the feature 👍🏼  Just add some styling and it's good to go!

File: src/components/card.js (hunk 1 above)
Comment (brandongallagher1999) [reply src/components/card.js:47]: Yep I'll add some premade button to make it look nice

**Comment by brandongallagher1999** [comment]
Made the change, and it looks nice now haha.

**Comment by brandongallagher1999** [comment]
https://imgur.com/a/YO53GUM

**Pull request merged**